src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -122,6 +122,62 @@ async function initializeConnection(options: CircuitStartOptions, logger: Logger
 
   Blazor._internal.forceCloseConnection = () => connection.stop();
 
+  Blazor._internal.sendJSDataStream = (data: ArrayBufferView, streamId: string, maximumIncomingBytes: number) => {
```

Comment:
Could this whole block be factored out into a separate file? The `Boot.*.ts` files risk being a general collection of all possible things otherwise, and it's hard to make sense of what happens in them :)